dependency-check-maven-config.xml

```diff
@@ -156,12 +166,27 @@
         <cve>CVE-2015-3415</cve>
         <cve>CVE-2015-3414</cve>
         <cve>CVE-2017-10989</cve>
+        <cve>CVE-2005-2992</cve>
+        <cve>CVE-2005-2945</cve>
+        <cve>CVE-2016-6153</cve>
+        <cve>CVE-2015-6607</cve>
+        <cve>CVE-2016-9878</cve>
+        <cve>CVE-2016-5007</cve>
+        <cve>CVE-2016-3674</cve>
     </suppress>
     <suppress>
         <notes>
             This CVE is incorrectly attributing a CXF vulnerability to Camel camel-cxf-2.19.0.jar.
```

Comment:
✏️  Nit: "These CVEs" instead of "This CVE"

Reply:
Fixed